alpha/template/composite/builder.go

```diff
@@ -196,21 +207,26 @@ func (rb *RawBuilder) Build(dir string, td TemplateDefinition) error {
 		return fmt.Errorf("raw template configuration is invalid: %s", strings.Join(validationErrs, ","))
 	}
 
-	// build the container command
-	containerCmd := exec.Command(rb.builderCfg.ContainerCfg.ContainerTool,
-		"run",
-		"--rm",
-		"-v",
-		fmt.Sprintf("%s:%s:Z", rb.builderCfg.CurrentDirectory, rb.builderCfg.ContainerCfg.WorkingDir),
-		"--entrypoint=cat", // This assumes that the `cat` command is available in the container -- Should we also build a `... render-template raw` command to ensure consistent operation? Does OPM already have a way to render a raw FBC?
-		rb.builderCfg.ContainerCfg.BaseImage,
-		path.Join(rb.builderCfg.ContainerCfg.WorkingDir, rawConfig.Input))
+	srcPath := path.Join(rb.builderCfg.InputDirectory, rawConfig.Input)
+
+	reader, err := os.Open(srcPath)
+	if err != nil {
+		return fmt.Errorf("error reading raw input file: %s, %v", srcPath, err)
+	}
+	defer reader.Close()
+
+	dcfg, err := declcfg.LoadReader(reader)
+	if err != nil {
+		return fmt.Errorf("error parsing raw input file: %s, %v", srcPath, err)
+	}
+
+	destPath := path.Join(dir, rawConfig.Output)
 
-	return build(containerCmd, path.Join(dir, rawConfig.Output), rb.builderCfg.OutputType)
+	return build(dcfg, destPath, rb.builderCfg.OutputType)
 }
 
 func (rb *RawBuilder) Validate(dir string) error {
-	return validate(rb.builderCfg.ContainerCfg, path.Join(rb.builderCfg.CurrentDirectory, dir))
+	return validate(rb.builderCfg.ContainerCfg, path.Join(rb.builderCfg.InputDirectory, dir))
```

Comment:
This namechange really illustrates how moving from container-in-container to container-less really moves us in the direction of re-examining the entire schema.  

Logically, this should be `builderCfg.ContainerCfg.WorkingDir` unless `CompositeConfig.Components[].Destination.Path` (dir here) is canonical, in which case `builderCfg.InputDirectory` is superfluous.   

I think we need to eliminate the `WorkingDir` element from the schema since it only really makes sense w/in a container.  

Similar for `InputDirectory` where we really need to be able to rely on the input path from the author. 

Still nibbling on this, but might just choose to leave as-is for this PR.

Reply:
On later thought, `WorkingDir` makes a lot of sense from the use-case from the maintainer of "I will look for contributions generated **here**".  

I think `InputDirectory` is still obsolete, though.